app/src/main/java/com/example/nockanakalinowej/View/TilesMatrixLayout.java

```diff
@@ -142,13 +141,14 @@ public void addImagesIDs(Map<Integer,Integer> IDs) {
     }
 
     public void tileOnclick(ImageButton clickedTile){
-        clickedTile.setBackgroundResource(R.drawable.tile_cover_selected);
+        clickedTile.setImageResource(R.drawable.images_tile_cover_selected);

```

Comment:
zbudowałeś to i działa, czy testujesz jenkinsa? bo analogiczne użycie jak IDs wyglądałoby tak:
R.images.tile_cover_selected